next-frontend/src/app/layout.tsx

```diff
@@ -25,9 +26,13 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
+    <html suppressHydrationWarning>
```

Comment:
why do we need this?

Reply:
Not a clever answer but: Because Chakra UI says so.
https://www.chakra-ui.com/docs/get-started/frameworks/next-app#setup-provider